cmd/utils/nodecmd/migrationcmd.go

```diff
@@ -138,6 +148,16 @@ func createDBConfigForMigration(ctx *cli.Context) (*database.DBConfig, *database
 			WriteCapacityUnits: ctx.Int64(utils.DstDynamoDBWriteCapacityFlag.Name),
 			PerfCheck:          !ctx.Bool(utils.DBNoPerformanceMetricsFlag.Name),
 		},
+
+		RocksDBConfig: &database.RocksDBConfig{
+			CacheSize:                 ctx.Uint64(utils.DstRocksDBCacheSizeFlag.Name),
+			DumpMallocStat:            ctx.Bool(utils.DstRocksDBDumpMallocStatFlag.Name),
+			DisableMetrics:            ctx.Bool(utils.DstRocksDBDisableMetricsFlag.Name),
+			Secondary:                 ctx.Bool(utils.DstRocksDBSecondaryFlag.Name),
+			CompressionType:           ctx.String(utils.DstRocksDBCompressionTypeFlag.Name),
+			BottommostCompressionType: ctx.String(utils.DstRocksDBBottommostCompressionTypeFlag.Name),
+			FilterPolicy:              ctx.String(utils.DstRocksDBFilterPolicyFlag.Name),
+		},
```

Comment:
Code similar to the one above is being repeated in the same pattern in many places. In the future, it is hoped that the db_manager side will set the default value as a flag value and be refactored so that only the options to be specially set are entered separately.